### PR TITLE
Add a partial for docs not updated for apps v2

### PR DESCRIPTION
**Pull request by andie787**
Was going to make some updates for the `primary_region` setting, but realized the affected docs need a more thorough review for apps V2 updates. This PR creates a partial for a banner. 

Banner:
Screenshot 2023-05-30 at 3 30 37 PM

### CODE REVIEW

File: app-guides/planetscale.html.md [hunk 1]
```diff
@@ -135,7 +135,7 @@ You'll be asked to give the app a name. Type a name using lowercase characters a
 
 You'll be prompted to pick an organization. Since every Fly user has a personal organization, let's pick that for now.
 
-You'll be asked for the region to deploy the first vm to. That will be the same value you set for `PRIMARY_REGION` above. For example `lhr`.
+You'll be asked for the region to deploy the first vm to. That will be the same value you set for `primary_region` above. For example, `lhr`.
```
Comment (catflydotio) [inline app-guides/planetscale.html.md:138]: If it's in `fly.toml` will you be asked?

File: languages-and-frameworks/crystal.html.md.erb [hunk 2]
```diff
@@ -196,10 +196,10 @@ Fly also supports multi-region deployments.
 
 To deploy to multiple regions, first [Create a PostgreSQL Cluster](/docs/postgres/), then follow these steps:
 
-1. Configure your primary region by setting the env `PRIMARY_REGION` in your fly.toml
+1. Configure your primary region by setting the `primary_region` in your fly.toml. The `fly deploy` command uses `primary_region` to determine where to create new Machines, as well as to set the `PRIMARY_REGION` environment variable within the Machines.
+
   ```toml
-  [env]
-    PRIMARY_REGION = "ord"
+  primary_region = "ord"
```
Comment (catflydotio) [inline languages-and-frameworks/crystal.html.md.erb:202]: Another case of an old config that's only doing an environment variable, not an app primary_region setting.

**Comment by catflydotio** [comment]
Lol. My piecewise comments seem to conclude we should put an "out-of-date" warning on top of both these files and a "remove-or-update" task on the board for them

File: app-guides/planetscale.html.md [hunk 3]
```diff
@@ -111,7 +111,7 @@ We've added a few files to make this sample app ready to run on Fly: a `Dockerfi
...
 1. Update the app's name at the top to one of your choice `app = "your-name-here"`.
 
-2. Update the region to the one nearest _your_ primary database: `PRIMARY_REGION = "lhr"`. We created _our_ database in `eu-west`. But Fly uses different region names than PlanetScale. And so you need to use the Fly region you would want your primary PlanetScale database to always be used from. As an example, let's say you have a PlanetScale database in _eu-west-1_. You plan on having a Fly app deployed to `lhr` and `iad`. You would set `PRIMARY_REGION = "lhr"` in the `fly.toml`. As that is the closest Fly region to the primary database's region.
+2. Update the region to the one nearest _your_ primary database: `primary_region = "lhr"`. We created _our_ database in `eu-west`. But Fly uses different region names than PlanetScale. And so you need to use the Fly region you would want your primary PlanetScale database to always be used from. As an example, let's say you have a PlanetScale database in _eu-west-1_. You plan on having a Fly app deployed to `lhr` and `iad`. You would set `primary_region = "lhr"` in the `fly.toml`, because that is the closest Fly region to the primary database's region.
```
[the remark this answers is not in the document]
Comment (andie787) [reply app-guides/planetscale.html.md:114]: Yep, I was making some assumptions. @catflydotio Should we have a "hasn't been updated for Apps V2 yet" banner?

**Pull request merged**